easybuild/easyconfigs/j/jemalloc/jemalloc-5.2.1-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'jemalloc'
+version = '5.2.1'
+
+homepage = 'http://jemalloc.net'
+description = """jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and
+ scalable concurrency support."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/jemalloc/jemalloc/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['ed51b0b37098af4ca6ed31c22324635263f8ad6471889e0592a9c0dba9136aea']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('binutils', '2.32'),
+]
+
+# From version 5.2.1 (or maybe earlier) it does no longer build,
+# nor try to install, documentation if xsltproc is missing.
+# So we can use normal installation.
+preconfigopts = "./autogen.sh && "
+configopts = "--with-version=%(version)s-0-g0000 "  # build with version info
+
+sanity_check_paths = {
+    'files': ['bin/jeprof', 'lib/libjemalloc.a', 'lib/libjemalloc_pic.a', 'lib/libjemalloc.%s' % SHLIB_EXT,
+              'include/jemalloc/jemalloc.h'],
+    'dirs': [],
+}
+
+# Why did someone set this?? LD_PRELOAD should NEVER be set automatically
```

Comment:
@akesandgren please update this comment?

Reply:
Update how?

I put it there to get peoples attention, like @boegel's since he is the one who put it there.
I think he said it would be ok to remove the LD_PRELOAD, but I'd like to hear from others it they rely on this for some reason...

Reply:
@akesandgren we use jemalloc with LD_PRELOAD for some programs, but it does not have to be set here, so for me it's ok to remove it.

the comment sounds a bit like pointing fingers to me, so maybe you can change the wording to make it a bit friendlier?

Reply:
Yeah, I think it makes sense to re-word this to make it sound a bit less aggressive (I don't mind, but others may frown when reading this).

How about this:

```python
# jemalloc can be used via $LD_PRELOAD, but we don't enable this by default, you need to opt-in to it